app/javascript/controllers/toggle_stop_route_controller.js

```diff
@@ -0,0 +1,23 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+    static targets = ["showOnStopsCheckbox", "visibleStopList", "visibleRouteList"];
+
+    connect() {
+        console.log('Connected to toggle_stop_route_controller');
```

Comment:
it doesn't have to happen right now, but you could simplify this down to just the `toggle()` method, and connect can call `toggle()`